client/models/section.coffee

```diff
@@ -45,4 +45,12 @@ module.exports = class Section extends Backbone.Model
     if @get('type') is 'artworks'
       _.extend super, ids: @artworks.pluck '_id'
     else
-      super
\ No newline at end of file
+      super
+
+   getLayout: ->
+    layout = 'column_width'
+    if @get('type') is 'text'
+      layout = 'overflow_fillwidth' if @get('body')?.includes('<blockquote>')
+    else if @get('layout')
+      layout = @get('layout')
+    return layout
```

Comment:
I wonder if we'll want an additional layout for blockquote since it's a little larger than the overflow_fillwidth for ImageCollection:

![image](https://user-images.githubusercontent.com/2236794/28849529-67a1a566-76e5-11e7-8f3c-4408061710ae.png)

This _could_ work if we adjust the width numbers of `overflow_fillwidth` per component, but I tend to feel like that ends up with a lot of CSS confusion especially with the way I'm building a new component in Reaction called `SectionContainer` that wraps every section component. SectionContainer knows about the following layout properties:

`column_width`has a max-width of 680px
`overflow_fillwidth` has 780px